memory/libhugetlbfs.py

```diff
@@ -67,9 +69,8 @@ def setUp(self):
             self.cancel("libpthread.a is required!!!"
                         "\nTry installing glibc-static")
 
-        self.page_sizes = ['16']
-        if 'POWER9' in cpuinfo and 'PowerNV' in cpuinfo:
-            self.page_sizes = ['2', '1024']
+        page_sizes = memory.get_supported_huge_pages_size()
+        self.page_sizes = [str(each // 1024) for each in page_sizes]
```

Comment:
Can't we do `// 1024` also in the utils ?

Reply:
@narasimhan-v I too thought about that, but the existing APIs in avocado return values in kB so I adhered to it.

Reply:
oh, then it makes sense.